src/stores/auth.js

```diff
@@ -146,5 +151,10 @@ export const useAuthStore = defineStore('auth', {
         throw new Error(err);
       }
     },
-  },
+    async oAuth(provider) {
+      const url = `http://${import.meta.env.VITE_API_URL}/api/user/${provider}/redirect`;
```

Comment:
We'll want to ensure we use https in prod, so we'll need to adjust things to include that in the API url I think i.e. set https:// in the value itself for prod and http:// for development